lib/macho/macho_file.rb

```diff
@@ -395,7 +395,8 @@ def get_load_commands
         # why do I do this? i don't like declaring constants below
         # classes, and i need them to resolve...
         klass = MachO.const_get "#{LC_STRUCTURES[cmd_sym]}"
-        command = klass.new_from_bin(@raw_data, endianness, offset, @raw_data.slice(offset, klass.bytesize))
+        view = MachOView.new(@raw_data, endianness, offset)
+        command = klass.new_from_bin(view, @raw_data.slice(offset, klass.bytesize))
```

Comment:
This looks a lot like the `@raw_data.slice(…)` should be in `LoadCommand#new_from_bin` instead of being passed as a second argument.